apps/wallets/smart-wallet/expo/.env.example

```diff
@@ -0,0 +1,3 @@
+EXPO_PUBLIC_CROSSMINT_API_KEY=
+EXPO_PUBLIC_CROSSMINT_JWT=
```

Comment:
`EXPO_PUBLIC_CROSSMINT_JWT` will be removed from the demo app envs once we get BYOS